src/sass/Detail.scss

```diff
@@ -1,18 +1,28 @@
+$detail-image-height: 420px;
+
 .detail {
   @extend .cont;
   width: 100%;
   min-width: 300px;
+  min-height: 650px;
   padding-top: 112px;
   display: flex;
   flex-direction: row;
   justify-content: space-between;
   .image {
     margin-right: 100px;
-    height: 420px;
+    height: $detail-image-height;
     background-size: contain;
     background-position: center center;
     background-repeat: no-repeat;
   }
+  .fa-motorcycle {
```

Comment:
No you don't :D it's a temporary solution for empty pre-load page, I talked with Malwa about some mockup in this case.